go.mod

```diff
@@ -12,6 +12,8 @@ replace github.com/filecoin-project/test-vectors => ./extern/test-vectors // pro
 
 replace github.com/filecoin-project/filecoin-ffi => ./extern/filecoin-ffi // provided via a git submodule
 
+replace github.com/filecoin-project/go-state-types => ./extern/go-state-types // provided via a git submodule
```

Comment:
you shouldn't have needed to use a git submodule for this, just `replace` to your fork and a commit on it

Reply:
`master` contains go-state-types@v0.16.0-rc1 now which has your change from there, so this isn't needed

Reply:
I'll replace it